ci/do_ci.sh

```diff
@@ -218,7 +221,9 @@ case $CI_TARGET in
         # toolchain is kept consistent. This ifdef is checked in
         # test/common/stats/stat_test_utility.cc when computing
         # Stats::TestUtil::MemoryTest::mode().
-        [[ "${ENVOY_BUILD_ARCH}" == "x86_64" ]] && BAZEL_BUILD_OPTIONS+=("--test_env=ENVOY_MEMORY_TEST_EXACT=true")
+        if [[ "${ENVOY_BUILD_ARCH}" == "x86_64" ]]; then
+            BAZEL_BUILD_OPTIONS+=("--test_env=ENVOY_MEMORY_TEST_EXACT=true")
+        fi
```

Comment:
👍 Like this esp. It much better now to read it.

Reply:
yeah me too - i think this is something we probably should have a policy about